src/styles/main.scss

```diff
@@ -1 +1,86 @@
-// put code here
+$hour: #00F;
+$second: #008001;
+$center: #FFA500;
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    height: 100%;
+}
+
+.container {
+
+    &__watch {
+        width: 80vmin;
+        height: 80vmin;
+       position: absolute;
+        border: 10px dotted black;
+        inset: 0;
+        margin: auto;
+        border-radius: 50%;
+    }
+
+    &__hour {
+        position: absolute;
+        left: 0;
+        right: 0;
+        margin: 0 auto;
+        width: 30px;
+        height: 100%;
+        z-index: 1;
+        animation: time 5s linear;
```

Comment:
The stopwatch should be animated infinitely and you can implement it very easily with css!